tests/zero-downtime-scaling/results/1.2.0/1.2.0.md

```diff
@@ -0,0 +1,530 @@
+Running 5m test @ http://cafe.example.com/coffee
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     1.44ms  802.80us  53.07ms   92.86%
+    Req/Sec   722.83     83.40     0.96k    71.09%
+  Latency Distribution
+     50%    1.31ms
+     75%    1.58ms
+     90%    1.96ms
+     99%    4.31ms
+  431719 requests in 5.00m, 153.98MB read
+  Socket errors: connect 0, read 20, write 0, timeout 0
+Requests/sec:   1438.61
+Transfer/sec:    525.42KB
+```
+
+HTTP Graph:
+
+![gradual-up-http.png](10-node/gradual-up-http.png)
+
+
+HTTPS wrk output
+
+```text
+wrk -t2 -c2 -d5m --latency --timeout 2s  https://cafe.example.com/tea
+Running 5m test @ https://cafe.example.com/tea
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     1.55ms    1.73ms 160.80ms   97.88%
+    Req/Sec   679.18     87.82     0.94k    69.26%
+  Latency Distribution
+     50%    1.38ms
+     75%    1.68ms
+     90%    2.09ms
+     99%    4.51ms
+  405615 requests in 5.00m, 141.96MB read
+  Socket errors: connect 0, read 19, write 0, timeout 0
+Requests/sec:   1351.90
+Transfer/sec:    484.51KB
+```
+
+HTTPS Graph:
+
+![gradual-up-https.png](10-node/gradual-up-https.png)
+
+Logs:
+
+- NGF error logs: none
+- NGINX error/warn logs: none
+- NGINX access logs
+  - 842986 [200 Oks]
+  - One 499 `INFO 2024-03-19T21:28:15.266352183Z [resource.labels.containerName: nginx] 10.150.0.27 - - [19/Mar/2024:21:28:15 +0000] "GET /tea HTTP/1.1" 499 0 "-" "-"`
+
+
+### Scale Down Gradually
+
+HTTP wrk output
+
+```text
+wrk -t2 -c2 -d20m --latency --timeout 2s  http://cafe.example.com/coffee
+Running 20m test @ http://cafe.example.com/coffee
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     8.99ms   71.48ms   1.07s    98.65%
+    Req/Sec   704.32     82.44     0.95k    74.78%
+  Latency Distribution
+     50%    1.35ms
+     75%    1.63ms
+     90%    2.03ms
+     99%  326.29ms
+  1603461 requests in 19.81m, 570.38MB read
+  Socket errors: connect 0, read 21, write 0, timeout 10
+Requests/sec:   1349.02
+Transfer/sec:    491.38KB
+```
+
+HTTP Graph
+
+![gradual-down-http.png](10-node/gradual-down-http.png)
+
+
+HTPPS wrk output
+
+```text
+wrk -t2 -c2 -d20m --latency --timeout 2s  https://cafe.example.com/tea
+Running 20m test @ https://cafe.example.com/tea
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     7.13ms   61.54ms   1.07s    98.98%
+    Req/Sec   661.77     77.50     0.90k    75.68%
+  Latency Distribution
+     50%    1.43ms
+     75%    1.73ms
+     90%    2.13ms
+     99%   85.50ms
+  1500047 requests in 19.75m, 523.58MB read
+  Socket errors: connect 0, read 27, write 0, timeout 12
+Requests/sec:   1265.79
+Transfer/sec:    452.41KB
+```
+
+HTTPS Graph
+
+![gradual-down-https.png](10-node/gradual-down-https.png)
+
+Logs:
+
+- NGF error logs: 25
```

Comment:
could you clarify what NGF errors do you see? here and in other similar places

Reply:
i do have some log files. I'll summarize with a sample error I've seen the most!

Reply:
I recommend including every unique error message with the count of how many times it happened.

Reply:
> I recommend including every unique error message with the count of how many times it happened.

+1 to @kate-osborn